offline/federation_preparation.md

```diff
@@ -0,0 +1,69 @@
+## RabbitMQ
+
+There are two ways to deploy the RabbitMQ cluster:
+
+Method 1
+---
+## Install RabbitMQ inside kubernetes cluster with the help of helm chart
+
+To install the rabbitmq,
+First copy the value and secret file:
+```
+cp ./values/rabbitmq/prod-values.example.yaml ./values/rabbitmq/values.yaml
+cp ./values/rabbitmq/prod-secrets.example.yaml ./values/rabbitmq/secrets.yaml
+```
+By default this will use ephemeral storage, to use the local persistance storage in kubernetes nodes, please refer to this doc - offline/local_persistance_storage.md
+
```

Comment:
```suggestion
By default this will create a RabbitMQ deployment with ephemeral storage. To use the local persistence storage of Kubernetes nodes, please refer to the related documentation in [offline/local_persistance_storage.md](./offline/local_persistance_storage.md).

```

If the filename contains the typo *persistance* (vs. *persistence*), too, it should be renamed.